spec/mongoid/extensions/date_class_mongoize_spec.rb

```diff
@@ -109,8 +109,12 @@
 
       context "when the string is an invalid time" do
 
+        let(:evolved) do
```

Comment:
This is a nit, but this variable should be `mongoized` not `evolved`

Reply:
done